data/xsd/phpdoc.xsd

```diff
@@ -36,7 +37,7 @@
             <xs:element name="version" type="pd:versionType" minOccurs="0" maxOccurs="unbounded"/>
             <xs:element name="template" type="pd:templateType" minOccurs="0" maxOccurs="unbounded"/>
         </xs:sequence>
-        <xs:attribute name="version" type="pd:version" use="required"/>
+        <xs:attribute name="v" type="pd:version" use="required"/>
```

Comment:
Yeah .. this is suboptimal..

The problem is that we have an element named version as well in the root of the XML document. And for XML that is not a problem but for loading the XML it is because attributes and elements are both transformed into array key => pairs. So version has, for the XML Loader a conflicting meaning

Reply:
I could not find a way to solve this with the XML Loader; so the easiest way was to rename one of the two. And I had a naming failure :D

Reply:
maybe rename it to `config_version`?

Reply:
Sounds good; for consistency, I will use configVersion. But that is the same in the end :)